packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -461,3 +483,67 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
     );
   }
 }
+
+/// Returns editing intent for selector from NSStandardKeyBindingResponding
```

Comment:
"Maps the selector from NSStandardKeyBindingResponding to the Intent if the selector is recognized."